web/src/test/java/org/springframework/security/web/authentication/www/BasicAuthenticationFilterTests.java

```diff
@@ -320,4 +320,105 @@ public void skippedOnErrorDispatch() throws Exception {
 
 		assertThat(response.getStatus()).isEqualTo(200);
 	}
+
+	@Test
+	public void testCredentialsWithUmlautUsingCharset_Utf_8() throws Exception {
```

Comment:
Although this test name follows the same pattern as other tests in this class, we would like any new tests to follow the convention *methodWhenConditionThenResult*.
A possible name could be:
```suggestion
	public void doFilterWhenTokenAndFilterCharsetMatchThenAuthenticated() throws Exception {
```

Reply:
I adjusted the code.